proxy.go

```diff
@@ -656,6 +657,21 @@ func (rp *reverseProxy) applyConfig(cfg *config.Config) error {
 		}
 	}
 
+	var customizedTransport http.RoundTripper = &http.Transport{
```

Comment:
you should put this part in the function `func newReverseProxy() *reverseProxy` that creates the reverse proxy so that you won't have to modify it afterward.
You can look at this code from an ongoing PR to see how to do it:
https://github.com/ContentSquare/chproxy/blob/993ff433f736a534e7b791dcf00e24ad73b7eb21/proxy.go#L52

Reply:
Ok.
Does it work upon config reload?

Reply:
I see, extra newReverseProxy call in applyConfig.
Will follow this pattern when https://github.com/ContentSquare/chproxy/pull/260 is merged.

Reply:
ok,https://github.com/ContentSquare/chproxy/pull/260 should be merged soon (I'm waiting for one last change that should happen today or tmr).